src/components/Modal/Modal.jsx

```diff
@@ -8,12 +8,14 @@ function Modal({ isVisible, onAction, title, children }) {
 			<div className={styles.contModal}>
 				<h1>{title}</h1>
 				{children}
-				<button
-					className={styles.closeModal}
-					onClick={onAction}
-					type="button"
-				>Cerrar
-				</button>
+				<div className={styles.buttons}>
+					<button
+						className={styles.closeModal}
+						onClick={onAction}
+						type="button"
+					>Cerrar
```

Comment:
Check the correct alignment
```suggestion
					>
					  Cerrar
```